pkg/api/v1/summary_test.go

```diff
@@ -0,0 +1,186 @@
+/*
+Copyright 2021 The TestGrid Authors.
```

Comment:
nit: 2023 right?

Reply:
those 2 years never happened, we are still in 2021 lol.

on a more serious note, done